themes/geekboot/layouts/partials/docs-navbar.html

```diff
@@ -75,9 +75,16 @@
             </li>
 
             <li class="nav-item col-xs p-2">
-              <a class="navbar-link" href="https://slack.crossplane.io" title="Join the Crossplane Slack" target="_blank" rel="noopener">
+              <div id="slack"><a class="navbar-link faa-pulse" href="https://slack.crossplane.io" title="Join the Crossplane Slack" target="_blank" rel="noopener">
                 {{ partialCached "icons/slack.svg" . }} <span class="icon-label">Crossplane Slack</span>
               </a>
+            </div>
+            {{ if .IsHome }}
+              <div id="tooltip" class="" role="tooltip">
+                Join the Crossplane Slack
```

Comment:
Could we consider making the text clickable? When I view it, I want to click on it. It feels like a call to action, but its weird that I can't click on it.

Reply:
Ask and you shall receive.